decentraland/unity-explorer
# chore: scene debug console app param + improve scene console toggling key binding

### WHY 

Currently creators can only debug their scenes during Local Scene Development (LSD), but they need a way of enabling the embedded scene debug console to read scene logs on deployed scenes.

Issue: https://github.com/decentraland/creator-hub/issues/238

### WHAT

* Renamed everything referring to local scene for the scene debug console
* Implemented 'scene-console' parameter check for enabling the scene console (and its report handling) based on it, besides LSD.
* Refactored the key binding to toggle the console to make it more reliable (had to add 2 input actions in the "Shortcuts" input actions map)

### QA INSTRUCTIONS

1. Download the build from this PR
2. Open the build using the new "scene-console" app parameter (winOS -> `"C:\Users\[YOUR-USER]\Downloads\Decentraland_windows64\Decentraland.exe" --scene-console` / macOS -> `open Decentraland.app --args --scene-console`)
3. After the world loads confirm that **you CAN open the top scene debug console** in Genesis Plaza (normally it's with the "`" key that's to the left of the "1" key in the keyboard, but depending on your keyboard a different special character key could be the toggling key)
4. Close that build and re-open the PR build **without the new parameter** (just double click it)
5. After the wolrd loads confirm that **you CANNOT open the top scene debug console**

https://github.com/user-attachments/assets/0018b8cc-3222-419a-85ac-94fc43e4abe5
